src/components/OperationPanes/SetOptions.js

```diff
@@ -65,7 +65,7 @@ export default function SetOptions(props) {
       label={
         <span>
           Set Flags{" "}
-          <HelpMark href="https://developers.stellar.org/docs/glossary/accounts/#flags" />
+          <HelpMark href="https://developers.stellar.org/docs/encyclopedia/flags" />
```

Comment:
This link might have changed. https://developers.stellar.org/docs/glossary#flags works